ydb/mvp/oidc_proxy/oidc_protected_page_nebius.h

```diff
@@ -0,0 +1,149 @@
+#pragma once
+
+#include "oidc_protected_page.h"
+
+namespace NMVP {
+
+class THandlerSessionServiceCheckNebius : public THandlerSessionServiceCheck {
+private:
+    using TBase = THandlerSessionServiceCheck;
+
+public:
+    THandlerSessionServiceCheckNebius(const NActors::TActorId& sender,
+                                const NHttp::THttpIncomingRequestPtr& request,
+                                const NActors::TActorId& httpProxyId,
+                                const TOpenIdConnectSettings& settings)
+        : THandlerSessionServiceCheck(sender, request, httpProxyId, settings)
+    {}
+
+    void StartOidcProcess(const NActors::TActorContext& ctx) override {
+        NHttp::THeaders headers(Request->Headers);
+        LOG_DEBUG_S(ctx, EService::MVP, TStringBuilder() << "Start OIDC process");
+
+        NHttp::TCookies cookies(headers.Get("Cookie"));
+        TString sessionToken = Base64Decode(cookies.Get(CreateNameSessionCookie(Settings.ClientId)));
```

Comment:
`TString Base64Decode(const TStringBuf s)` will ignore error in case if an input string is not decodable from base64:

Reply:
changed to Base64StrictDecode